src/journal-remote/journal-upload.c

```diff
@@ -614,29 +631,16 @@ static int perform_upload(Uploader *u) {
                 return log_error_errno(SYNTHETIC_ERRNO(EIO),
                                        "Upload to %s failed with code %ld: %s",
                                        u->url, status, strna(u->answer));
-        else if (status < 200)
+        if (status < 200)
                 return log_error_errno(SYNTHETIC_ERRNO(EIO),
                                        "Upload to %s finished with unexpected code %ld: %s",
                                        u->url, status, strna(u->answer));
-        else {
 #if LIBCURL_VERSION_NUM >= 0x075300
```

Comment:
the ifdef needs to move too?

Reply:
Indeed. Not necessary, but it is better. I will move it. Thanks.